extensions/amp-form/0.1/test/test-amp-form.js

```diff
@@ -330,7 +331,9 @@ describes.repeated('', {
         emailInput.setAttribute('required', '');
         form.appendChild(emailInput);
         sandbox.spy(form, 'checkValidity');
-        sandbox.stub(ampForm.xhr_, 'fetch').returns(Promise.resolve());
+        const {promise, stub} = stubWithCalledPromise(
```

Comment:
polling on `callCount` shouldn't cause flakiness only impact would be promise getting resolved up to 8ms later than the call time. `poll` also takes an optional timeout (defaults to 1600ms) which means in the case of test failure (waiting for a stub call that never happens) it will timeout independent of the test timing out which provides more information on where test failure happened.